.github/workflows/test_and_deploy.yml

```diff
@@ -0,0 +1,161 @@
+name: Tests, Package, and Deployment
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+    branches: [next, qa, main]
+    tags: ["v*"]
+  release:
+    types: [published]
```

Comment:
should we remove this as in https://github.com/neutrons/quicknxs/pull/180, since we will create tag + release at the same time?